pkg/kotsadmsnapshot/backup.go

```diff
@@ -930,3 +946,59 @@ func prepareIncludedNamespaces(namespaces []string) []string {
 	}
 	return includedNamespaces
 }
+
+// excludeShutdownPodsFromBackup will exclude pods that are in a shutdown state from the backup
+// this is to prevent the hook backup from failing if a pod is in a shutdown state and cannot be backed up
+func excludeShutdownPodsFromBackup(ctx context.Context, clientset kubernetes.Interface, backup *velerov1.Backup) (err error) {
```

Comment:
i think there's an edge case in the minimal rbac scenario if `*` is part of the included namespaces because the `prepareIncludedNamespaces` would remove all the namespaces and keep `*`, and this whole function would be a no-op. one workaround would be to execute this function before the `prepareIncludedNamespaces` function is called. if you do that, please add a comment.